jobs/adjust_import.py

```diff
@@ -0,0 +1,55 @@
+import click
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import isnull, col, lit, udf
+
+
+@click.command()
+@click.option("--salt", required=True)
+@click.option("--iterations", default=1000)
+@click.option("--klen", default=32)
+@click.option("--project", required=True)
+@click.option("--input_table", required=True)
+@click.option("--output_table", required=True)
+@click.option("--bucket", required=True)
+def main(
+    salt,
+    iterations,
+    klen,
+    project,
+    input_table,
+    output_table,
+    bucket,
+):
+    spark = (SparkSession
+        .builder
+        .appName("adjust_gps_hash")
+        .getOrCreate())
+
+    @udf("string")
+    def pbkdf2_sha1hmac(msg, salt, iterations, klen):
+        import hashlib
+        import base64
+        return base64.b64encode(
+            hashlib.pbkdf2_hmac('sha1', str.encode(msg), str.encode(salt), iterations, klen)
```

Comment:
Why not `sha256` and 100k iterations, as per https://docs.python.org/3/library/hashlib.html#hashlib.pbkdf2_hmac?

Reply:
This is actually to [match code in data sent from Fenix.](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/components/metrics/MetricsUtils.kt#L116)